File: FragmentsApp/app/src/main/java/com/example/fragmentsapp/secondFragment.java
package com.example.fragmentsapp;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.TextView;
import android.widget.Toast;

/**
 * A simple {@link Fragment} subclass.
 * Use the {@link secondFragment#newInstance} factory method to
 * create an instance of this fragment.
 */
public class secondFragment extends Fragment {
    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_second, container,false);
        Button btn = view.findViewById(R.id.secondBtn);
        TextView txt = view.findViewById(R.id.title2);

        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(getActivity(), "Good Byye", Toast.LENGTH_SHORT).show();
            }
        });
        return view;
    }
}

File: FragmentDataSharing/app/src/main/java/com/tiramisu/fragmentdatasharing/SecondFragment.kt
package com.tiramisu.fragmentdatasharing

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView

class SecondFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        val view : View = inflater.inflate(R.layout.fragment_second, container, false)
        val name : TextView = view.findViewById(R.id.textViewName)
        val userName = arguments?.getString("username").toString()
        name.text = userName

        return view
    }


}